systemd/system/boot-splash-start.service

```diff
@@ -0,0 +1,26 @@
+#
+#  boot-splash-start.service
```

Comment:
Any chance this and the feature file could be moved to `kano-splash`?

Reply:
It would more naturally fit there to be honest, but then the image files would have to move too.
And they're quite big.

Reply:
I see your point that the usage of kano-splash in the OS is not part of the tool itself, but it is perhaps at least specific to the repo itself or another repo altogether. Happy with this for now 👍